apps/angular/1-projection/src/styles.scss

```diff
@@ -1,3 +1,7 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+.bg-light-green {
+  background-color: rgba(0, 250, 0, 0.1);
+}
```

Comment:
you should not put anything in the global stylesheet. It will get too big. 
Try to put everything at component level.